source/Calamari.Tests/KubernetesFixtures/ResourceStatus/ResourceUpdateReporterTests.cs

```diff
@@ -102,6 +110,7 @@ public void ReportsRemovedResourcesCorrectly()
                 {
                     new KeyValuePair<string, string>("name", "redis"),
                     new KeyValuePair<string, string>("removed", bool.TrueString),
+                    new KeyValuePair<string, string>("countOfChecks", "1"),
```

Comment:
Is there a way we can get a test to produce a second update to make sure we're getting a count of 2?

Reply:
(or more than 2 if you want)

Reply:
Because the method that gets tested simply put whatever the number is passed in into the service message, we can't test the incrementation of the number here. The current test structure captures what is happening during one check but not the whole cycle.
It would be nice if we add some checks to the incrementation of the numbers etc. Because we are considering changing the logic of stabilisation timeout and execution timeout, I think it is better to add the test in after the logic is re-implemented, which will make writing such tests much easier.